toolbox/__init__.py

```diff
@@ -93,6 +107,17 @@ def func():
         # UMAP legend
         self.ui.clear_button.clicked.connect(self.clear_utterances)
 
+    def set_current_wav(self, index):
+        self.current_wav = self.waves_list[index]
+
+    def save_last_wav(self):
+        if not self.current_wav is None:
+            self.ui.save_audio_file(self.current_wav, Synthesizer.sample_rate)
+
+    def replay_last_wav(self):
+        if not self.current_wav is None:
```

Comment:
Same as above, suggest disabling replay button when there is no wave.

Reply:
For both cases, this was just an extra check. I just removed it on the last commit